### ISSUE TITLE
Commit your project to Git

### ISSUE THREAD

**Issue by russellw999** [SAPDocuments/Tutorials#3277]
Tutorial URL: https://developers.sap.com/tutorials/pre-teched-2018-webide-git.html

Please specify the step you are referring to

On Developer.sap.com he image files do not render on this tutorial, i have tested it on Chrome and Firefox on 2 separate machines.

It appears that this tutorial is superseded by the more recent one where the images do render
12/02/2018  Commit your 	Project files to SAP Cloud Platform Git Repository
https://developers.sap.com/tutorials/hcp-webide-commit-git.html

in summary there are 2 tutorial doing showing the same thing on developer,sap.com, but this older one does not have images which render

**Comment by firfarmer** [comment]
Image links have been broken recently in multiple tutorials.  Kind of embarrassing for a company like SAP.

**Comment by thecodester** [comment]
The pre-teched tutorial has been removed now. If there are issues with other tutorials feel free to open an issue. Sorry for the problems.
